src/ReactWrapper.js

```diff
@@ -286,6 +286,17 @@ export default class ReactWrapper {
   }
 
   /**
+   * Returns the HTML of the node.
+   *
+   * NOTE: can only be called on a wrapper of a single node.
+   *
+   * @returns {String}
+   */
+  html() {
+    return this.single(n => findDOMNode(n).outerHTML.replace(/\sdata-reactid+="[^"]+"/g, ''));
```

Comment:
if this is returning an actual dom node, would it make more sense to `.cloneNode()` it, and then `removeAttribute('data-reactid')`, and *then* `.outerHTML`?